addons/web/static/src/core/install_prompt/install_prompt.scss

```diff
@@ -0,0 +1,30 @@
+.o_install_prompt {
+    border-radius: 1em;
```

Comment:
Use class `rounded-X` instead ?

Reply:
I used this to use an "appleish" look, to make it look like a native popup.
We only have up to rounded-3, which is more squared and looks weird to match native popup style.

That's why I prefered to use scss instead of a bootstrap class. I can use another value